survey/templates/survey/survey_response.html

```diff
@@ -8,7 +9,9 @@
 {{ csrf | json_script:"csrf" }}
 {{ value | json_script:"responseValue" }}
 {{ survey.survey_config | json_script:"configData" }}
-<script type="module" src="{% static 'js/sort-ui/sort_survey_response.js' %}"></script>
```

Comment:
Can we override the inbuilt `static` template tag?

https://github.com/django/django/blob/aed303aff57ac990894b6354af001b0e8ea55f71/django/templatetags/static.py#L174-L179